src/fsharp/TastOps.fs

```diff
@@ -1729,7 +1731,10 @@ let isStructTy g ty =
     | Some tcref -> 
         let tycon = tcref.Deref
         tycon.IsStructRecordOrUnionTycon || tycon.IsStructOrEnumTycon
-    | _ -> false
```

Comment:
we can probably just use `isStructTupleTy` here instead.

Reply:
Thanks, Will, just testing that now.